WcaOnRails/app/javascript/edit-schedule/SchedulesEditor/fullcalendar.js

```diff
@@ -46,7 +46,14 @@ export function generateCalendar(eventFetcher, showModalAction, scheduleWcif, ad
     eventResizeStart: fullCalendarHandlers.onResizeStart,
     eventResizeStop: fullCalendarHandlers.onResizeStop,
     eventResize: fullCalendarHandlers.onSizeChanged,
-    select: (start, end) => fullCalendarHandlers.onTimeframeSelected(showModalAction, start, end),
+
+    // We need to set selectMinDistance greater than 0 to suppress `select`
+    // events when the user simply single clicks (and does not drag) on the
+    // calendar. See https://fullcalendar.io/docs/selectMinDistance.
+    // I have no idea why this isn't the default behavior.
+    selectMinDistance: 5,
+    select: (start, end, jsEvent, view) => fullCalendarHandlers.onTimeframeSelected(showModalAction, start, end),
```

Comment:
Since we don't use `jsEvent` and `view` I'd just leave them out (same for the function below).

Reply:
Oh whoops! Good catch. Those were leftover from when I was poking around trying to figure out how to do this.